src/themes/components/box.ts

```diff
@@ -0,0 +1,11 @@
+import type { SystemStyleObject } from '@chakra-ui/styled-system';
+
+const baseStyle: SystemStyleObject = {
```

Comment:
I don't think we wanna do that for the box component as it will quite alternate the default behaviour when thinking of a div. We have the "Center" component that can be wrapped to achieve that behaviour. 
There will be overall more use cases where we need the box as a block element. Making it displayed as a flex element will alternate that behaviour quite a bit

Reply:
I completely removed the file, is it fine?